client-hc/src/main/java/com/graphhopper/api/GraphHopperWeb.java

```diff
@@ -125,12 +124,6 @@ public OkHttpClient getDownloader() {
         return downloader;
     }
 
-    @Override
-    public boolean load(String serviceUrl) {
-        this.routeServiceUrl = serviceUrl;
-        return true;
-    }
-
```

Comment:
Any concerns about backward-compatiblity here? client-hc users now need to set the url via the constructor, but this will become apparent immediately, because there will be a compile error.

Reply:
Not a problem, I think.